cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -51,6 +53,9 @@ func init() {
 	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.nodeIP, "node-ip", "",
 		"nodeIP is the IP that should be used as the node object's IP. "+
 			"If unset, kubelet will determine the IP itself.")
+	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.clusterDNS, "cluster-dns", "",
+		"The DNS server IP passed to kubelet, that will be used to configure all containers for DNS resolution "+
+			"instead of the host's DNS server. The latter, is the default kubelet configuration.")
```

Comment:
You should call out what happens when it is not specified similar to the `node-ip`.

>The DNS server IP passed to kubelet, that will be used to configure all containers for DNS resolution. If unset, kubelet will determine the DNS server to use.

The determination should be left to the kubelet